src/api/storage-pools.tsx

```diff
@@ -77,9 +105,9 @@ const getClusterPool = (
   pool: Partial<LxdStoragePool>,
 ): Partial<LxdStoragePool> => {
   const poolForCluster = { ...pool };
-  if (pool.driver !== cephDriver) {
-    delete poolForCluster.config;
-  }
```

Comment:
The ceph case is removed here, but not handled anymore

Reply:
ah yes this was deliberate. The logic from before was keeping the config object for a ceph cluster pool creation because there are no member specific configs for ceph, with the current approach we are basically doing the same so this is no longer needed.

Reply:
Right, everything not in memberSpecificConfigKeys is a cluster wide config option. Makes sense.